backend/Ript-Fitness-Backend/src/main/java/com/riptFitness/Ript_Fitness_Backend/web/dto/ExerciseDto.java

```diff
@@ -0,0 +1,72 @@
+package com.riptFitness.Ript_Fitness_Backend.web.dto;
+
+import com.riptFitness.Ript_Fitness_Backend.domain.model.AccountsModel;
+
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+
+public class ExerciseDto {
+	
+	
+	public Long exerciseId;
+	public int sets;
+	public int reps;
+	// private int weight; Not sure what to do with this yet; may use later.
+	public String nameOfExercise;
+	
+	// Reference to the users account (AccountsModel)
+	public AccountsModel account; // Reference to the account that owns this exercise
+	
+	// We also will need a reference to the workout model: (UNCOMMENT BELOW ONCE CHRIS IS DONE)
+	//public WorkoutModel workout;
+	
+	// Getters and Setters
+    public Long getExerciseId() {
```

Comment:
Similar to the data model, getters and setters aren't necessary for public variables